testing/catalog.go

```diff
@@ -572,9 +692,8 @@ func (c *Catalog) AnnotatedPod(name string, annotations map[string]string) corev
 
 // Sleep1hPodSpec defines a simple pod that sleeps 60*60s for testing
 func (c *Catalog) Sleep1hPodSpec() corev1.PodSpec {
-	one := int64(1)
 	return corev1.PodSpec{
-		TerminationGracePeriodSeconds: &one,
+		TerminationGracePeriodSeconds: helper.Int64(1),
```

Comment:
We should replace all occurrences with the helper, not just this one.

Reply:
👍

Reply:
https://www.pivotaltracker.com/story/show/164729001